### ISSUE TITLE
onChange sometimes stop working

### ISSUE THREAD

**Issue by wilav-dev** [zenoamaro/react-quill#843]
I have react quill field inside a form. It's working fine but when I leave a tab in the browser and after some time (I'm not sure if after some time or after mac sleep mode, or anything else), onChange is not calling anymore while typing unless some specific keys or actions (Eg. spacebar as in my video example, delete key, selecting text and setting format such as bold but only using keyboard)
If I open a new tab, it starts working again, but if I refresh the same tab that is not working, it is still not working.

https://user-images.githubusercontent.com/71793326/195986539-67c1d3f0-cfce-49a7-84ce-994c5a199710.mov

### Ticket due diligence

- [x] I have verified that the issue persists under ReactQuill `v2.0.0-beta.4`

### ReactQuill version
- v2.0.0-beta.4

**Comment by raosudheesh** [comment]
Hey @wilav-dev , had the same problem. Close that particular tab and open it again. Worked fine for me, may be some browser caching issues

**Comment by sachin717** [comment]
Hey there ,
i am facing the same issue is there any update on that
please elaborate me if possible

**Comment by mattch** [comment]
I'm seeing the same issue. @raosudheesh 's suggestion seems to solve it.  Though I'm surprised a hard refresh (cmd-R) doesn't work, and the tab has to be closed.  I'm really curious why. Any idea?

**Comment by Somacros** [comment]
This issue is still happening, any solutions?

**Comment by raosudheesh** [comment]
Did you try closing the tab and opening it again?

**Comment by Somacros** [comment]
Yes, and it starts working if I select another page (that react-quill input is in a global component) but if I return to the exact route where the problem was happening, the issue appears again.
